ios/FluentUI/Vnext/Avatar/Avatar.swift

```diff
@@ -153,6 +70,33 @@ class MSFAvatarStateImpl: NSObject, ObservableObject, MSFAvatarState {
     }
 }
 
+// MARK: - PersonaData MSFAvatarStateImpl extension
+
+extension MSFAvatarStateImpl {
+    convenience init(style: MSFAvatarStyle, size: MSFAvatarSize, personaData: PersonaData) {
+        self.init(style: style, size: size)
+        self.copyProperties(from: personaData)
+    }
+
+    func copyProperties(from personaData: PersonaData) {
+        if let name = personaData.composedName {
+            self.primaryText = name.0
+            self.secondaryText = name.1
+        } else {
+            let identifier = (personaData.name.count > 0) ? personaData.name : personaData.email
+            self.primaryText = identifier
+        }
+
+        self.foregroundColor = personaData.color
+        self.image = personaData.image
+        self.imageBasedRingColor = personaData.customBorderImage
+        self.isOutOfOffice = personaData.presence == .outOfOffice
+        self.hasRingInnerGap = !personaData.hideInsideGapForBorder
+        self.presence = MSFAvatarPresence(personaData.presence)
+        self.ringColor = personaData.color
+    }
+}
+
```

Comment:
Just noticed this: It seems like your previous change in the avatar (addition of the **.renderingMode(.original)** modifier) broke the rendering of the icon images:

![Screen Shot 2021-06-23 at 10 11 51 AM](https://user-images.githubusercontent.com/68076145/123139789-6e963a80-d40b-11eb-9723-f338556265f5.png)

Reply:
I'll open a bug and fix that in a separate PR, thanks for catching

Reply:
Created a new PR: 

**Fix `Avatar` image rendering mode** https://github.com/microsoft/fluentui-apple/pull/617

Reply:
Thank you!